testing/i3blocks/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Marvin Steadfast <marvin@xsteadfastx.org>
+# Maintainer: Marvin Steadfast <marvin@xsteadfastx.org>
+pkgname=i3blocks
+pkgver=20190321_git
+pkgrel=0
+pkgdesc="A minimalist scheduler for your status line scripts"
+url="https://github.com/vivien/i3blocks"
+arch="all"
+license="GPL"
+makedepends="autoconf automake"
+subpackages="$pkgname-doc"
+_commit="ec050e79ad8489a6f8deb37d4c20ab10729c25c3"
+source="$pkgname-$pkgver.tar.gz::https://github.com/vivien/i3blocks/archive/$_commit.tar.gz"
+builddir="$srcdir/$pkgname-$_commit"
+
+build() {
+	cd "$builddir"
+	./autogen.sh
+	./configure
+	make
+}
+
+check() {
+	cd "$builddir"
+	make check
+}
+
+doc() {
+	cd "$builddir"
+	gzip docs/i3blocks.1
+	install -Dm644 docs/i3blocks.1.gz "$subpkgdir"/usr/share/man/man1/i3blocks.1.gz
+}
```

Comment:
compressing the manpage is handled by `abuild` during `rootpkg`, please remove `doc()`